python-sdk/tutorials/map_expansion_tutorial.ipynb

```diff
@@ -239,14 +243,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "nusc_map.render_next_roads(x, y)"
+    "nusc_map.render_next_roads(x, y, figsize=1)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Working with Lanes"
+    "### Working with Lanes\n",
+    "For the prediction challenge we added connectivity information to the map expansion (v1.2) to efficiently query which lane is connected to which other lanes. Below we render the lane and lane_connectivity objects. The lanes and lane_connectors are defined by parametric curves. The `resolution_meters` parameter specifies the discretization resolution of the curve. If we set it to a high value (e.g. 100), the curves will appear as straight lines."
```

Comment:
i'm having trouble understanding the figure, so are the lane connectors colored red here? and lanes are varying colors?

Reply:
We should suggest that it is never larger than a few meters or so. There can only be confusion if set higher.

Reply:
@venice-erin Yes everything is rendered and each arcline path has a random different color to tell it apart.
@emwolff I agree.